.gitattributes

```diff
@@ -1,14 +1,7 @@
-/.appveyor.yml export-ignore
-/.gitattributes export-ignore
-/.github export-ignore
-/.gitignore export-ignore
-/.travis.yml export-ignore
-/build.properties export-ignore
-/build.xml export-ignore
-/composer.lock export-ignore
-/docs export-ignore
-/phpcs.xml.dist export-ignore
-/phpstan.neon.dist export-ignore
-/phpunit.xml.dist export-ignore
-/run-all.sh export-ignore
-/tests export-ignore
+/* export-ignore
+/bin -export-ignore
+/bin/** -export-ignore
```

Comment:
Are there two expressions per directory required? What happens if we remove the first one?

Reply:
Yes it is required, there will  not be a `bin` directory if you include the first one, because that's equivalent to saying `bin` is ignored for the export, although every file in it is isn't. That's at least what I observed when I tried.